### DISCUSSION TITLE
How do I replace my original search with this one?

### DISCUSSION THREAD

**Discussion by wezlol** [RevertIT/mybb-rt_livesearch#8] [Q&A]
title

**Comment by RevertIT** [comment]
This is supplementary search addition for fast searching. It should not replace/restrict the `/search.php` as many features are stripped down.

**Reply by wezlol** [reply]
My bad, i meant the search in the upper right corner

**Reply by RevertIT** [reply]
https://github.com/RevertIT/mybb-rt_livesearch?tab=readme-ov-file#-how-to-add-custom-search-box
